src/providers/MeetingProvider/types.ts

```diff
@@ -19,12 +17,14 @@ export enum DevicePermissionStatus {
   DENIED = 'DENIED',
 }
 
-export interface MeetingJoinData {
-  meetingInfo: any;
-  attendeeInfo: any;
+export interface MeetingManagerJoinOptions {
```

Comment:
nit: This should be in `MeetingManager.tsx`

Reply:
This is correct here, MeetingManager is just a plain class and not a component right?

Reply:
Oh right, that's my bad, it's in `types.ts` which is correct. Thought it was in `MeetingProvider.tsx`